src/dmd/dsymbolsem.d

```diff
@@ -3319,9 +3319,9 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
                     // if the first parameter has a default argument, then the rest does as well
                     if (ctd.storage_class & STC.disable)
                     {
-                        ctd.deprecation("@disable'd constructor cannot have default "~
+                        ctd.deprecation("`@disable`'d constructor cannot have default "~
```

Comment:
Does @disable'd highlight really work here? Maybe could be rephrased to keep it.

Reply:
Yeah, it's odd.  I reworded it.  Also because it calls the `.deprecated` member function it already fills in the "constructor x.y.this()`, so there's no need to redundantly repeat it in the error message.  There are many other error messages like that, but that's another PR.